### ISSUE TITLE
Ed25519 certificates cause SSLyze to crash. 

### ISSUE THREAD

**Issue by JarrahG** [nabla-c0d3/sslyze#444]
**Describe the bug**
sslyze fails when scanning an Ed25519 Certificate. It looks like the certificate is not recognised, resulting in a "None" object. 

**To Reproduce**
1. Attempt to scan a server using an Ed25519 certificate. (--regular was used)
2. View the following error message:
```python
Traceback (most recent call last):
  File "/home/user/.local/bin/sslyze", line 11, in <module>
    load_entry_point('sslyze==3.0.6', 'console_scripts', 'sslyze')()
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/__main__.py", line 76, in main
    output_hub.server_scan_completed(scan_result)
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/cli/output_hub.py", line 53, in server_scan_completed
    out_generator.server_scan_completed(server_scan_result)
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/cli/console_output.py", line 75, in server_scan_completed
    for line in cli_connector_cls.result_to_console_output(scan_command_result):
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/plugins/certificate_info/_cli_connector.py", line 93, in result_to_console_output
    result_as_txt.extend(cls._cert_deployment_to_console_output(index, cert_deployment))
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/plugins/certificate_info/_cli_connector.py", line 106, in _cert_deployment_to_console_output
    deployment_as_txt.extend(cls._get_basic_certificate_text(leaf_certificate))
  File "/home/user/.local/lib/python3.8/site-packages/sslyze/plugins/certificate_info/_cli_connector.py", line 281, in _get_basic_certificate_text
    cls._format_field("Signature Algorithm:", certificate.signature_hash_algorithm.name),
AttributeError: 'NoneType' object has no attribute 'name'
```

Server being tested is a golang net/http server with a CA signed Ed25519 cert. 

**Expected behavior**
Regular output of SSLyze scan. This occurs on other scans. 

**Python environment (please complete the following information):**
 - OS: [e.g. Windows 10, Ubuntu 16.04, macOS Sierra]
 - Python version: [e.g. 3.6, 3.7]
OS: Fedora 32
Python version: 3.8.3
SSLyze version: 3.0.6 from pip.

**Comment by nabla-c0d3** [comment]
Released in v3.0.7.

**Comment by JarrahG** [comment]
Thank you for creating such a handy tool and for fixing this.
